docker/gp_tests/scripts/configs/partial_restore_test_config.json

```diff
@@ -0,0 +1,2 @@
+"WALE_S3_PREFIX": "s3://gppartialbucket",
```

Comment:
I would add `database` so it would be easier to distinguish these two tests. Like this: `docker/gp_tests/scripts/configs/partial_database_restore_test_config.json`